sources/packages/backend/apps/workers/src/controllers/assessment/_tests_/e2e/assessment.controller-verifyAssessmentCalculationOrder.e2e-spec.ts

```diff
@@ -533,6 +545,12 @@ describe("AssessmentController(e2e)-verifyAssessmentCalculationOrder", () => {
       programYearTotalPartTimeCSGP: 8,
       programYearTotalPartTimeSBSD: 12,
       programYearTotalPartTimeBCAG: 14,
+      programYearTotalPartTimeBooksAndSuppliesCost: 0,
```

Comment:
Can we have a second application with a different offering intensity to ensure the part-time/full-time total will be retrieved?
It can be part of an upcoming PR if one is planned to be raised with more E2E tests.